screens/TabThreeScreen.tsx

```diff
@@ -1,37 +1,36 @@
-import React, {useState} from 'react';
+import React, { useState } from 'react'
 import {
   SafeAreaView,
   StyleSheet,
   TextInput,
   View,
   Text,
   ScrollView,
-} from 'react-native';
-import TodoInsert from '../components/TodoInsert';
-import TodoList from '../components/TodoList';
+} from 'react-native'
+import TodoInsert from '../components/TodoInsert'
+import TodoList from '../components/TodoList'
 
 const App = () => {
+  const [todos, setTodos] = useState<any>([])
```

Comment:
Nit: should provide type for the `todos` array. From the code below, it should be something like 
```
{
  id: string;
  textValue: string,
  checked: boolean,
}[]
```